package.json

```diff
@@ -33,25 +33,25 @@
   "homepage": "https://github.com/testing-library/user-event#readme",
   "devDependencies": {
     "@babel/cli": "^7.8.4",
-    "@babel/core": "^7.9.0",
-    "@babel/preset-env": "^7.9.5",
+    "@babel/core": "^7.9.6",
+    "@babel/preset-env": "^7.9.6",
     "@babel/preset-react": "^7.9.4",
-    "@testing-library/dom": "^7.2.1",
-    "@testing-library/jest-dom": "^5.5.0",
+    "@testing-library/dom": "^7.5.1",
+    "@testing-library/jest-dom": "^5.7.0",
     "@testing-library/react": "^9.4.0",
     "@testing-library/vue": "^4.1.0",
-    "all-contributors-cli": "^6.14.1",
+    "all-contributors-cli": "^6.14.2",
     "babel-core": "7.0.0-bridge.0",
-    "babel-jest": "^25.3.0",
+    "babel-jest": "^26.0.1",
     "codecov": "^3.6.5",
     "husky": "^4.2.5",
-    "jest": "24.9.0",
-    "lint-staged": "^10.1.3",
-    "prettier": "^2.0.4",
+    "jest": "26.0.1",
+    "lint-staged": "^10.2.2",
+    "prettier": "^2.0.5",
     "react": "^16.13.1",
     "react-dom": "^16.13.1",
     "regenerator-runtime": "^0.13.5",
-    "semantic-release": "^17.0.4",
+    "semantic-release": "^17.0.7",
     "travis-deploy-once": "5.0.11",
     "vue": "^2.6.11",
     "vue-template-compiler": "^2.6.11"
```

Comment:
Was it necessary to update all dependencies for this PR?

Reply:
No need, but I like to keep dependencies up to date. I can revert rhat if you'll like that.

Reply:
ping @kentcdodds